src/scimlfunctions.jl

```diff
@@ -4073,6 +4065,15 @@ function BatchIntegralFunction(f, integrand_prototype; kwargs...)
     BatchIntegralFunction{true}(f, integrand_prototype; kwargs...)
 end
 
+########## Utility functions
+
+function sys_or_symbolcache(sys, syms, paramsyms, indepsym = nothing)
+  if sys === nothing && (syms !== nothing || paramsyms !== nothing || indepsym !== nothing)
+    sys = SymbolCache(syms, paramsyms, indepsym)
+  end
+  return sys
+end
```

Comment:
The `syms` stuff is deprecated, I don't get why this is still staying around. If anything it should throw a big warning not to use it.

Reply:
Should I remove the keywords from the constructors or just `@warn`?

Reply:
I guess `@warn` since it was accidentally documented, and remove it from the docstrings.

Reply:
Fixed in https://github.com/SciML/SciMLBase.jl/pull/603/commits/1f88d7fd6b88844d55cfdf91ff3971b31d9897ac